maquetacion/scss/_tech.scss

```diff
@@ -0,0 +1,47 @@
+@import 'variables';
+@import 'colors';
+
+.content-cards {
+  margin-bottom: #{$text-medium * 2};
+  padding: $text-medium #{$text-medium * 2};
+}
+
+.card {
+  background-color: $green;
+  border-radius: $card-border-radius;
+  max-width: 100%;
+  padding: $text-medium;
+  position: relative;
+  width: $card-width;
+
+  &::before {
+    content: '';
+    border: 2px solid $green;
+    border-radius: $card-border-radius;
+    height: 100%;
+    left: -#{$text-small};
+    position: absolute;
+    top: -#{$text-small};
+    width: 100%;
+    z-index: -1;
+  }
+
+  @at-root #{&}-image {
```

Comment:
You dont need this things... It would be better if the CSS is simple as posible.